src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptyIsFalse.java

```diff
@@ -71,8 +69,7 @@ int test1(int badFormat1,int badFormat2,
     {
         return 0;
     }
-
-    /** method that is 20 lines long **/
+     /** method that is 20 lines long **/
```

Comment:
```suggestion
    /** method that is 20 lines long **/
```
FYI: its a little off, imho. But format is in the examples not enforced imho.